source/base/mpi.cc

```diff
@@ -1033,6 +1033,32 @@ namespace Utilities
     }
 
 
+    namespace internal
+    {
+      template <typename T>
+      T
+      builtin_single_broadcast(const MPI_Comm &   comm,
+                               const T &          object_to_send,
```

Comment:
```suggestion
                               T           object_to_send,
```
`MPI_Bcast` stores the result in its first parameter and copying the types for which this function is instantiated is cheap.

Reply:
Fixed in 0e21a8a